app/repositories/ApprovalRepository.scala

```diff
@@ -33,7 +33,7 @@ import scala.concurrent.Future
 
 trait ApprovalRepository {
   def update(process: ApprovalProcess): Future[RequestOutcome[String]]
-  def getById(id: String): Future[RequestOutcome[JsObject]]
+  def getById(id: String): Future[RequestOutcome[ApprovalProcess]]
```

Comment:
Comment, possibly to discuss. The inital external-guidance repository get methods returned JsObjects whatever the potential object type leaving the transtition to a concrete type to manage microservice. We now seem to creating the objects in the repository only for them to be converted back to json when returned to Manage?

Reply:
Some of the new code needs to get the approval process so it can use the meta data we have created against it (not ocelot meta).
Have had different conversations about where conversion should be done but if done in controller means service methods can make use of the other data stored in DB from repository, not just the process.  I think it's strange that the repo returns the ocelot process and not the actual object it has stored - my thinking is repo should be dumb to what data is actually required, unless doing a particular projection.